server/src/auth/auth.service.ts

```diff
@@ -16,14 +22,68 @@ export class AuthService {
 
   /** Get identifier from a google token */
   getIdFromGoogleToken(token: string): string | undefined {
-    return undefined;
+
+    return undefined
+  }
+
+  
+  async verifyGoogle(token_id : string): Promise<any> {
+    try {
```

Comment:
The stuff in this method should be in getIdFromGoogleToken. It returns undefined if no id could be found/google/verifyIdToken fails, or it returns the id (that's all we need).